src/pages/index.js

```diff
@@ -1,21 +1,93 @@
-import React from 'react'
-import { Link } from 'gatsby'
+/* eslint-disable array-bracket-spacing */
+import React, { useState, useEffect } from 'react'
+import { Paper } from '@material-ui/core'
+import Grid from '@material-ui/core/Grid'
+import { useMediaQuery } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import CircularProgress from '@material-ui/core/CircularProgress'
+import DirectorCard from '../components/DirectorCard/DirectorCard'
+import getQueryDataDirectors from '../directors/getQueryDataDirectors'
+import { useTranslation } from 'react-i18next'
```

Comment:
It's better to separate local and npm packages

Reply:
done